src/sentry/monitors/consumers/monitor_consumer.py

```diff
@@ -181,7 +197,7 @@ def _process_message(wrapper: Dict) -> None:
                     )
                     logger.debug(
                         "check-in guid %s already associated with %s not payload %s",
-                        params["check_in_id"],
+                        check_in_id,
```

Comment:
same comment with casting here, but in reverse

Reply:
This will get cast to string no problem